README.md

```diff
@@ -133,7 +116,97 @@ include the `restPort` configuration to tell the exporter which port to use.
+Metrics will then be available from `<application-root>/metrics`.
+
+
+## Downloading the release
+
+You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).
+
+To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script, which is provided with each release and also can be downloaded from the Releases page, and then run:
+
+```
+bash getXXX.sh <your-config-file>
+```
+
+## Building from source
+
+Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
+is the Maven-assigned version number.
+
+Adding `-Dconfiguration=<some-config-file>` will insert the specified configuration as its default and remove
+the version number to simplify deployment to WebLogic Server.
+
+# Sidecar
+
+## Build and run
+
+The sidecar implementation must be built with JDK11 or later. Building the project with JDK8 will skip the sidecar module.
+
+After building, execute:
+```
+java -jar wls-exporter-sidecar/target/wls-exporter-sidecar.jar
+```
+
+This will start the exporter on port 8080, and expect to find a WebLogic Server instance locally, listening on port 7001.
+
+That may be changed by specifying parameters on the command line:
+
+Setting | Default | Property
+------------ | ------------- | -------------
+Domain name | (use WLS definition) | DOMAIN
+Exporter port | 8080 | EXPORTER_PORT
+WebLogic host | localhost | WLS_HOST
+WebLogic port | 7001 | WLS_PORT
+Use https | false | WLS_SECURE
+
+## Configure the exporter
+
+The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
+You can do this with `curl`
+
+```
+curl -X PUT -i -u myname:mypassword \
+    -H "content-type: application/yaml" \
+    --data-binary "@<path to yaml>" \
+    http://localhost:8080/configuration
+``` 
+
+where you should replace `myname` and `mypassword` with the credentials expected by WebLogic Server for its REST API,
+and `<path to yaml>` is the relative path to the configuration to use.
+
+## Access the metrics
+
+Once the exporter is configured, a GET to `http://localhost:8080/metrics` (or whatever port was chosen) will return the current metrics.
+
+## Building a docker image
+
+If docker is installed, running 
+
+```
+docker build . -t <image-name>
+```
+
+will build the project and create a docker image with the specified name. It is not necessary even 
+to do the Maven build first, as that will
+happen as part of creating the image. When running behind a firewall, it is necessary to specify a value for 
+the MAVEN_OPTS variable on the command line. For example
+
+```
+docker build . --build-arg MAVEN_OPTS="-Dhttps.proxyHost=www-proxy -Dhttps.proxyPort=80" -t <image-name>
+```
+
+To allow the Maven image to download the dependencies. 
```

Comment:
To allow the Maven image to download the dependencies. -> ??? Not a sentence. If you are completing your previous thought, then change this to -> This allows the Maven image to download the dependencies.

Reply:
Oh, here's the sidecar Docker image build command :). Maybe this was just an expectation of where this command would be... It's somewhat hard to read the original intermixed with the comments :).